src/evaluator.ts

```diff
@@ -152,17 +152,31 @@ export class Evaluator {
           return true;
         }
 
-        // Should Target be included via segment rules
-        if (
-          segment.rules &&
-          (await this.evaluateClauses(segment.rules, target))
-        ) {
-          this.log.debug(
-            'Target %s included in segment %s via rules\n',
-            target.name,
-            segment.name,
+        if (segment.servingRules && segment.servingRules.length > 0) {
```

Comment:
Fyi, in Typescript when using the `length` property of an array to check if it's not empty, you can simply truthy check it and leave out the > 0.  But both work and it's a matter of style choice!